db-migrations/000016_remove_duplicate_ip_assignments.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS backup_aws_private_ip_assignment;
```

Comment:
question: should I add an insert statement here to get the records back into the original table before dropping this?

Reply:
(I am guessing yes but want to be sure of approach here before going deeper on it)

Reply:
yep, that works.

Reply:
TODO: insert records back into table from the backup table before dropping it